recipe/conda_build_config.yaml

```diff
@@ -4,6 +4,47 @@ c_stdlib_version:  # [osx and x86_64]
 MACOSX_SDK_VERSION:         # [osx]
   - 12.3                    # [osx]
 
+# Needs higher stdlib for CUDNN9
+# CUDA 12 on aarch needs newer glibc, which requires us to override
+# the whole monster zip just to bump c_stdlib_version / docker_image
+c_stdlib_version:                                   # [linux and aarch64]
+  - 2.17                                            # [linux and aarch64]
+  - 2.17                                            # [linux and aarch64]
+  - 2.28                                            # [linux and aarch64]
+docker_image:                                       # [linux and aarch64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64      # [linux and aarch64]
+  - quay.io/condaforge/linux-anvil-cuda:11.8        # [linux and aarch64]
+  # TODO: We're using the aarch64-cuda:11.8 image, based on ubi8, only to get a
+  # newer glibc for the CUDA 12 builds. For some reason, aarch64 + CUDA 12 is
+  # expecting GLIBC_2.27.
+  - quay.io/condaforge/linux-anvil-aarch64-cuda:11.8  # [linux and aarch64] # glibc 2.28
```

Comment:
@h-vetinari can you please confirm that this is OK? I copied this over from CUDNN since the previous one failed at detecting a working compiler.

Reply:
Think this should be

```suggestion
  - quay.io/condaforge/linux-anvil-alma-x86_64:8  # [linux and aarch64]
```

Reply:
![image](https://github.com/user-attachments/assets/e5d38e38-533a-4326-b3b5-2ed7a3a58610)

The thing is we tried it before

/home/conda/feedstock_root/build_artifacts/libtorch_1727921957520/_build_env/bin/../lib/gcc/aarch64-conda-linux-gnu/12.4.0/../../../../aarch64-conda-linux-gnu/bin/ld: cannot represent machine `i386:x86-64'

It may have been due to the linker script, so lets try again.